src/services/postApi.ts

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+
+const API_BASE_URL = 'http://localhost:8080/api/post';
+
+export const createPost = async (formData: FormData) => {
+    const response = await axios.post(API_BASE_URL, formData, {
+        headers: {
+            'Content-Type': 'multipart/form-data',
+        },
+    });
+    return response.data;
+};
+
+export const getPostById = async (id: bigint) => {
```

Comment:
Let's not use bigint, I don't think it's very conventional or necessary. We can safely use `number` as a type for integers.